### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by bamarni**
* Update dcos-launch download URL
* Wait for DNS records to be propagated in package tests
* Use https when adding 1.7 universe in tests
* Use helloworld instead of marathon in a test for `pacakge describe`

### CODE REVIEW

**Comment by bamarni** [comment]
run all integration tests

**Comment by bamarni** [comment]
run all integration tests

**Comment by bamarni** [comment]
run all integration tests

**Comment by bamarni** [comment]
run all integration tests

**Comment by bamarni** [comment]
run all integration tests

File: cli/dcoscli/test/package.py [hunk 1]
```diff
@@ -48,30 +52,25 @@ def package(package_name, deploy=False, args=[]):
 
 def setup_universe_server():
     # add both Unvierse servers with static packages
-    add_app('tests/data/universe-v3-stub.json', True)
-    add_app('tests/data/helloworld-v3-stub.json', True)
+    add_app('tests/data/universe-v3-stub.json', False)
+    add_app('tests/data/helloworld-v3-stub.json', False)
```
Comment (klueska) [inline cli/dcoscli/test/package.py]: What does False do here instead of True?

File: cli/dcoscli/test/package.py [hunk 2]
```diff
@@ -48,30 +52,25 @@ def package(package_name, deploy=False, args=[]):
...
 
     assert_command(
         ['dcos', 'package', 'repo', 'remove', 'Universe'])
 
+    watch_all_deployments()
+
     # Add the two test repos to Cosmos
     for name, url in UNIVERSE_TEST_REPOS.items():
-        assert_command(['dcos', 'package', 'repo', 'add', name, url])
+        # wait for DNS records for the universe app to be propagated
```
Comment (klueska) [inline cli/dcoscli/test/package.py]: This looks like a completely different strategy from before for waiting for the universe to come online. What's better about this way (or different at least)?

File: cli/tests/integrations/test_package.py [hunk 3]
```diff
@@ -84,7 +84,7 @@ def test_repo_list():
         assert_command(['dcos', 'package', 'repo', 'list'], stdout=repo_list)
 
 
-def test_repo_add():
+def test_repo_add_and_remove():
```
Comment (klueska) [inline cli/tests/integrations/test_package.py]: Why do you combine these now?

File: cli/dcoscli/test/package.py (hunk 1 above)
Comment (bamarni) [reply cli/dcoscli/test/package.py]: it proceeds without watching for deployments, this is not needed as there is a `watch_all_deployments()` below.

File: cli/dcoscli/test/package.py (hunk 2 above)
Comment (bamarni) [reply cli/dcoscli/test/package.py]: `pacakge repo add` now fails if the universe service is unreachable, see see https://mesosphere.slack.com/archives/C02H2798B/p1512426126000076

File: cli/tests/integrations/test_package.py (hunk 3 above)
Comment (bamarni) [reply cli/tests/integrations/test_package.py]: they were order-dependent, cf. the commit message.

**Comment by bamarni** [comment]
run all integration tests

**Comment by bamarni** [comment]
@takirala @jsancio : I'd appreciate your confirmation on this (cf. commit messages for detailed descriptions), seems like a new version of cosmos has been released lately into dcos and a few tests began to fail.

**Comment by bamarni** [comment]
run all integration tests

**Pull request merged**